pkg/subctl/cmd/validate_deployment.go

```diff
@@ -145,6 +146,10 @@ func checkPods(item restConfig, submariner *v1alpha1.Submariner, operatorNamespa
 		}
 	}
 
+	if !checkPodsStatus(kubeClientSet, operatorNamespace) {
+		return
```

Comment:
I have a suggestion for later PRs

Let's not stop if something fails, so we can get full diagnostic from the other remaining items.